src/audio/AudioStream.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function AudioStream( listener ) {
+
+	this.context = listener.context;
+
+	this.gain = this.context.createGain();
+	this.gain.connect( listener.getInput() );
+
+}
+
+Object.assign( AudioStream.prototype, {
+
+	getOutput: function () {
+
+		return this.gain;
+
+	},
+
+	setAudioElement: function ( audioElement ) {
```

Comment:
I'm probably missing something but... Wouldn't it be simpler to just add this method to [Audio](https://github.com/mrdoob/three.js/blob/dev/src/audio/Audio.js)?